### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
tendrl-bug-id: Tendrl/performance-monitoring#220
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
Verified
@r0h4n @nthomas-redhat @shtripat Please review

**Comment by r0h4n** [comment]
Need an issue for this

File: tendrl/performance_monitoring/sds/__init__.py [hunk 1]
```diff
@@ -402,7 +402,7 @@ def configure_monitoring(self, integration_id):
             )
             return
         for plugin in SDSPlugin.plugins:
-            if plugin.name == sds_tendrl_context['sds_name']:
+            if plugin.name == sds_tendrl_context.get('sds_name', ''):
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/__init__.py]: sds_name should be mandatory, ain't it?

Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/__init__.py]: I saw a traceback in the very first scan probably when the details were still being loaded as in https://github.com/Tendrl/performance-monitoring/issues/220#issue-240326689 point 2
So added this for safety...

**Comment by anmolbabu** [comment]
@r0h4n @nthomas-redhat @shtripat Please review

**Comment by r0h4n** [comment]
Please send this PR on "master" branch

**Pull request merged**